docs/source/azure/policy/resources/networkinterface.rst

```diff
@@ -42,4 +42,16 @@ and then perform the delete operation on those ready for deletion.
           - type: marked-for-op
             op: delete
         actions:
-          - type: delete
\ No newline at end of file
+          - type: delete
+
+.. code-block:: yaml
+
+    policies:
```

Comment:
In samples (or even tests) you might consider just looking for `addressPrefix` or `nextHopType` as I think these make more sense to people and are what they would be trying to filter on.   e.g. `addressPrefix != 10.0.3.0/24` or `Does Not Contain Route where nextHopType==VirtualNetworkGateway`

Reply:
Yeah makes sense, that was one of the test policies I made when I was working on this. I'll switch the test and sample to check `VirtualNetworkGateway`